packages/observability/mock-data/import-mock-data.sh

```
@@ -0,0 +1,397 @@
+
+# Default configuration
+INPUT_FILE="mock-metrics.txt"
+METHOD="remote-write"
+PROMETHEUS_URL=""
+BEARER_TOKEN=""
+PROMETHEUS_NAMESPACE="opendatahub"
+PROMETHEUS_SERVICE="prometheus-data-science-monitoringstack"
+PROMETHEUS_PORT="9090"
+USE_THANOS=false
+DRY_RUN=false
+BATCH_SIZE=500
```

Shellcheck (0.11.0)

[warning] BATCH_SIZE appears unused. Verify use (or export if used externally).

(SC2034)